apps/openmw/mwworld/store.cpp

```diff
@@ -1031,16 +1064,20 @@ namespace MWWorld
     //=========================================================================
 
     template <>
```

Comment:
Now that name handling is in the record classes, we should be able to get rid of the Store specializations for ESM::Script and ESM::StartScript.